mobile/src/Offers.js

```diff
@@ -17,212 +17,186 @@
 import React, { Component } from 'react'
 import ReactNative, { TouchableOpacity, Text, View, Image, Dimensions, Linking, Platform } from 'react-native'
 import client, { Color } from '@doubledutch/rn-client'
-import Chevron from './Chevron'
+
+const aspectRatio = 1.6
 
 export default class Offers extends Component {
   constructor() {
     super()
     this.state = {
       renderText: false,
-      renderConfirm: false
+      renderConfirm: false,
+      height: 0,
+      width: 0
     }
   }
+  componentDidMount = () => {
+    Image.getSize(this.props.image, (width, height) => {
```

Comment:
It looks like this is getting the pixel size of the image and using that directly, rather than the aspect ratio.  Does this mean it will not scale the image to the width of the card?